src/utils/constants.mjs

```diff
@@ -0,0 +1 @@
+export const RESTAKE_USER_AGENT = 'ReStake'
```

Comment:
```suggestion
export const RESTAKE_USER_AGENT = 'REStake'
```

Reply:
@tombeynon I also thought it'd be nice to include the version/commit or something, but restake doesn't have versioning, right?

Reply:
I was thinking the same, sadly not. The Git versions are good but the app doesn't know about them. I'll raise an issue to get that added.

Reply:
Node's package.json is also nice, if doing it via package.json version we can just import it as json file, so getting the version would be pretty straightforward.
Also suggest looking into libraries that are making GitHub releases and stuff, I use goreleaser for all of the Golang tools, guess there should be something similar in JS ecosystem. One solution I found by briefly googling is semantic-release.